src/app/views/discover/home/_fs-post/fs-post.ts

```diff
@@ -0,0 +1,79 @@
+import Vue from 'vue';
+import { Component, Emit, Prop } from 'vue-property-decorator';
+import { FiresidePost } from '../../../../../_common/fireside/post/post-model';
+import { AppImgResponsive } from '../../../../../_common/img/responsive/responsive';
+import AppMediaItemBackdrop from '../../../../../_common/media-item/backdrop/backdrop.vue';
+import AppUserAvatarImg from '../../../../../_common/user/user-avatar/img/img.vue';
+import {
+	getVideoPlayerFromSources,
+	VideoPlayerController,
+} from '../../../../../_common/video/player/controller';
+import AppVideo from '../../../../../_common/video/video.vue';
+import AppCommunityOverlayPill from '../_community-overlay-pill/community-overlay-pill.vue';
+
+@Component({
+	components: {
+		AppMediaItemBackdrop,
+		AppImgResponsive,
+		AppVideo,
+		AppUserAvatarImg,
+		AppCommunityOverlayPill,
+	},
+})
+export default class AppHomeFsPost extends Vue {
+	@Prop({ type: FiresidePost, required: true })
+	post!: FiresidePost;
+
+	videoController: VideoPlayerController | null = null;
+	imgLoaded = false;
+
+	@Emit('loaded')
+	emitLoaded() {}
+
+	get mediaItem() {
+		if (this.post.hasVideo) {
+			return this.post.videos[0].posterMediaItem!;
+		}
+
+		return this.post.media[0];
+	}
+
+	get video() {
+		if (!this.post.hasVideo) {
+			return;
+		}
+
+		return this.post.videos[0];
+	}
+
+	get bgStyle() {
+		return {
+			backgroundImage: `url('${this.mediaItem.mediaserver_url}')`,
+		};
+	}
+
+	mounted() {
+		if (this.video?.postCardVideo) {
+			this.videoController = new VideoPlayerController(this.video.postCardVideo, null);
+
+			this.videoController.volume = 0;
+			this.videoController.muted = true;
```

Comment:
Would it be preferred to pass in a `VideoPlayerControllerContext` other than `null` here, either using `'gif'` or creating a different one that stays muted?